lib/utils/standalone.js

```diff
@@ -31,12 +31,16 @@ const arch = (() => {
 
 module.exports = {
   resolveLatestTag: async () => {
-    const { body } = await got(
+    const response = await fetch(
       isInChina
         ? 'https://sls-standalone-sv-1300963013.cos.na-siliconvalley.myqcloud.com/latest-tag'
         : 'https://api.github.com/repos/serverless/serverless/releases/latest'
     );
-    return isInChina ? body : JSON.parse(body).tag_name;
+    if (isInChina) {
+      return response.body;
```

Comment:
I think you're looking to return just the tag name as a string here, and that in this case, the response body is expected to be just that tag name? In which case, you still need to consume the stream here with `response.text()`

Reply:
@griffithsbs just update this part as well.